python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -244,7 +256,19 @@ def _task_metadata(self):
         cluster = response.get("Cluster")
         task_arn = response.get("TaskARN")
 
-        task = self.ecs.describe_tasks(tasks=[task_arn], cluster=cluster)["tasks"][0]
+        def describe_task_or_raise(task_arn, cluster):
+            return self.ecs.describe_tasks(tasks=[task_arn], cluster=cluster)["tasks"][0]
+
+        try:
+            task = backoff(
+                describe_task_or_raise,
```

Comment:
This could be a lambda instead but I thought it was marginally more readable with the function.